### DISCUSSION TITLE
Feedback & The Future of the Silicon Salon

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#86] [Silicon Salon]
Tell us what you thought of the first Silicon Salon! Was it too long? Too short? Did it cover what you wanted to discuss or not?

And, what should we do in the future? More Salons like this one? Somewhat different? Shorter, but more regular office hours?

**Comment by ChristopherA** [comment]
In addition to additional virtual events like our Silicon Salon today, there are also opportunities to meet and collaborate together F2F.

In particular, I'm the founder & host of [Rebooting Web of Trust](https://www.weboftrust.info) (twitter: [@rwotevents](https://twitter.com/rwotevents)), which this fall will be in The Hague, Netherlands, from September 26-30th [eventbrite details](https://www.eventbrite.com/e/rebooting-the-web-of-trust-xi-2022-the-hague-tickets-347605426187).

#RWOT is a collaborative event where together we together create papers, standards, and reference code to advance decentralized identity. Out of our workshops have come standards like W3C Verifiable Credentials, W3C Decentralized Identifiers, and over 50 collaborative white papers.

Part of the way the process works is that people submit "topic papers" in advance of the workshop for everyone to consider. Maybe our community can prepare a topic paper on the requirements for new chips or hardware security in emerging digital identity technology? Or a topic papers about risks, threats, and best practices for biometrics and and privacy?

-- Christopher Allen

**Comment by ChristopherA** [comment]
There has been some discussion with the [project & sustaining sponsors](https://www.blockchaincommons.com/sponsors.html) of Blockchain Commons to hold another Silicon Salon, possibly in mid August, specifically to continue the topic of [boot, firmware & supply chain](https://www.siliconsalon.info/#3-boot-firmware--supply-chain) (see also [post-evenbt discussion](https://github.com/BlockchainCommons/Airgapped-Wallet-Community/discussions/90)).

If you are interested in participating, presenting, or [sponsoring](https://github.com/sponsors/blockchaincommons) at our next Silicon Salon on this topic, [let me know](mailto:ChristopherA@LifeWithAlacrity.com)!
